### PR TITLE
(CAT-1143)-Conversion_of_erb_templates_to_epp

### PR DESCRIPTION
**Pull request by praj1001**
## Summary
Conversion of ERB To EPP for templates in docker module.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

**Review by Ramesh7 — APPROVED** [review]
LGTM

can we get one more review before we merge, as its big PR?

**Review by LukasAud — APPROVED** [review]
LGTM

**Pull request merged**
